examples/layers/widgets/spinning_mixed.dart

```diff
@@ -54,7 +54,7 @@ void attachWidgetTreeToRenderTree(RenderProxyBox container) {
                     ElevatedButton(
                       child: Row(
                         children: <Widget>[
-                          Image.network('https://flutter.dev/images/favicon.png'),
+                          Image.network('https://storage.googleapis.com/cms-storage-bucket/4fd0db61df0567c0f352.png'),
```

Comment:
Was this change intentional? Just curious, why is it necessary?

Reply:
`https://flutter.dev/images/favicon.png` 404s and thus made the example unusable. `https://storage.googleapis.com/cms-storage-bucket/4fd0db61df0567c0f352.png` is the current favicon we use for flutter.dev - so I replaced it with that. Not ideal since this can get out of date again, but not sure what stable image we could link to from here other wise...

Reply:
I vaguely remember seeing a `flutter_assets` package with images and videos - is that something we can use here? That's a complete short in the dark, if it's not viable perhaps we could use `https://docs.flutter.dev/assets/images/shared/brand/flutter/icon/64.png` instead as that URL seems a bit more "official".

Reply:
Changed it to `https://docs.flutter.dev/assets/images/shared/brand/flutter/icon/64.png` for now.

Reply:
I wonder if we could just replace this with the `FlutterLogo` widget. Let me try that.